engine/class_modules/priest/sc_priest.cpp

```diff
@@ -629,6 +643,8 @@ void init()
   expansion::bfa::register_leyshocks_trigger( 193223, STAT_VERSATILITY_RATING );
   // Dark Ascension
   expansion::bfa::register_leyshocks_trigger( 280711, STAT_VERSATILITY_RATING );
+  // Apotheosis
+  expansion::bfa::register_leyshocks_trigger( 200183, STAT_VERSATILITY_RATING);
```

Comment:
super nit picky, but can you add an extra space before paren end on these?

i.e.
```
( 200183, STAT_VERSATILITY_RATING );
```